src/viur/core/modules/file.py

```diff
@@ -658,6 +661,15 @@ def create_src_set(
         if not file:
             return ""
 
+        if isinstance(file, LanguageWrapper):
+            current_lang = current.language.get()
+            if language in file and file[language]:
+                file = file[language]
+            elif current_lang and current_lang in file:
+                file = file[current_lang]
+            else:
+                file = file[conf.i18n.default_language]
```

Comment:
This should satisfy already, and returns empty when no language is given, but it is a LanguageWrapper.

```suggestion
            language = language or current.language.get()
            if not language or not (file := file.get(language)):
                return ""
```

The additional check on `conf.i18n.default_language` is already handled by `current.language.get()`, as this ContextVar is initialized with `conf.i18n.default_language` on request processing.